react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Redirect} from 'react-router-dom';
+import styles from '../../styles.module.scss';
+import Navbar from "../Navbar"
+import { connect } from "react-redux";
+import actionCreators from "../../../../../redux/actions";
+
+class Matches extends Component {
+  
+  async componentDidMount(){
+    const response = this.props.dispatch(actionCreators.getMatches());
+    //this.props.isLoading = true;
+    //const response = await TicTacToeService.getMatches();
+    if(response.ok){
+      this.props.matches = response.data;
+      this.props.isLoading = false;
```

Comment:
You can't change props.
Props are passed by the parent (or hoc) but can't be changed.